index.php

```diff
@@ -2,190 +2,205 @@
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="applylocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/apply-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/apply-global-2.html"); ?>
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-1.html"); ?>
+				<!-- TemplateBeginEditable name="givelocal" -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/local/give-local.html"); ?>
+				<!-- TemplateEndEditable -->
+				<?php wdnInclude("/wdn/templates_5.0/includes/global/give-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/idm.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/search.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/header-global-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-1.html"); ?>
+	<!-- TemplateBeginEditable name="affiliation" -->
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-affiliation-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-1.html"); ?>
+	<!-- TemplateBeginEditable name="titlegraphic" -->
+	<a class="unl-site-title-medium" href="/">VITA Lincoln</a>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/site-title-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/logo-lockup-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-group.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-1.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-toggle-btn.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-child-1.html"); ?>
+	<!-- TemplateBeginEditable name="navlinks" -->
+		<?php include "$root/sharedcode/navigation.php"; ?>
+	<!-- TemplateEndEditable -->
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-child-2.html"); ?>
+		<?php wdnInclude("/wdn/templates_5.0/includes/global/nav-menu-2.html"); ?>
+</header>
+
+<main class="dcf-main" id="dcf-main" role="main" tabindex="-1">
+
+	<!-- TemplateBeginEditable name="hero" -->
+	<div class="dcf-hero dcf-hero-default dcf-sr-only">
+		<!-- TemplateEndEditable -->
+		<div class="dcf-hero-group-1">
+			<div class="dcf-breadcrumbs-wrapper">
+				<nav class="dcf-breadcrumbs" id="dcf-breadcrumbs" role="navigation" aria-label="breadcrumbs">
+					<!-- TemplateBeginEditable name="breadcrumbs" -->
+					<ol>
+						<li><a href="https://www.unl.edu/">Nebraska</a></li>
+						<li><a href="/">VITA Lincoln</a></li>
+						<li><span aria-current="page">Home</span></li>
```

Comment:
I really appreciate the use of `aria-current` here. One step closer to being accessible--now just the rest of the site haha 😬